hw/xbox/nv2a.c

```diff
@@ -3221,6 +3224,12 @@ static void pgraph_method(NV2AState *d,
 
         /* I guess this kicks it off? */
         if (image_blit->operation == NV09F_SET_OPERATION_SRCCOPY) {
+
+            char message[256];
+            sprintf(message,"NV2A: NV09F_SET_OPERATION_SRCCOPY");
+            glEnable(GL_DEBUG_OUTPUT); //FIXME: Elsewhere
+            glDebugMessageInsert(GL_DEBUG_SOURCE_APPLICATION, GL_DEBUG_TYPE_MARKER, 0, GL_DEBUG_SEVERITY_NOTIFICATION, strlen(message), message);
```

Comment:
Will do. Macro GL_DPRINTF(fmt,...) maybe so we can include line information?
GL_DGROUP_BEGIN(fmt,...) + GL_DGROUP_END() also?